DependencyInjection/DoctrinePHPCRExtension.php

```diff
@@ -212,7 +212,7 @@ private function loadJackalopeSession(array $session, ContainerBuilder $containe
         $backendParameters += $session['backend']['parameters'];
         // only set this default here when we know we are jackalope
         if (!isset($backendParameters['jackalope.check_login_on_server'])) {
-            $backendParameters['jackalope.check_login_on_server'] = $container->getParameter('kernel.debug');
+            $backendParameters['jackalope.check_login_on_server'] = false;
```

Comment:
Can't we set this default value in the Configuration class? I don't like such logic in the extension class.